demo/Nonlinear Noisy Pendulum.ipynb

```diff
@@ -0,0 +1,1404 @@
+{
```

Comment:
<p>Free energy also encompasses the complexity term, not only the accuracy term</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/biaslab/ReactiveMP.jl/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
@abpolym